### PR TITLE
Add django-hijack and dependencies (attempt 2)

### PR DESCRIPTION
**Pull request by risicle**

### CODE REVIEW

**Comment by risicle** [comment]
(just added licenses)

**Comment by aneeshusa** [comment]
Can you please use HTTPS for all of the pypi.python.org links? Refs #13809.

**Comment by risicle** [comment]
@aneeshusa done.

**Comment by risicle** [comment]
(updated django_hijack to 2.0.7 - no point in it already being out of date)

**Comment by joachifm** [comment]
This looks good to me, are you ready to merge?

**Comment by risicle** [comment]
Hmm hold on I might have got the sha256 wrong for django-hijack-2.0.7.tar.gz - what are the rest of you getting for it?

File: pkgs/top-level/python-packages.nix [hunk 1]
```diff
@@ -8062,6 +8084,45 @@ in modules // {
     };
   };
 
+  django_hijack = buildPythonPackage rec {
+    name = "django-hijack-${version}";
+    version = "2.0.7";
+
+    src = pkgs.fetchurl {
+      url = "https://pypi.python.org/packages/source/d/django-hijack/${name}.tar.gz";
+      sha256 = "1jj9f51j32gqy8xfn97x4qkwxkhcpbvwynrwd4nkd0375886f4kz";
```
Comment (joachifm) [inline pkgs/top-level/python-packages.nix]:
```
$ nix-prefetch-url https://pypi.python.org/packages/source/d/django-hijack/django-hijack-2.0.7.tar.gz
path is ‘/nix/store/g38xnjhckvw7iab072via93ypqvncchy-django-hijack-2.0.7.tar.gz’
0rpi1bkfx74xfbb2nk874kfdra1jcqp2vzky1r3z7zidlc9kah04
```

**Comment by risicle** [comment]
Yup - agree with you now - don't know where I got the other hash from. Ready for merge provided the tests pass.

**Pull request merged**